### DISCUSSION TITLE
Love the plugin, hope it'll be updated for Zot 7

### DISCUSSION THREAD

**Discussion by mechphoenix** [tefkah/zotero-night#135] [General]
Hey, I really love this dark mode plugin, it's got intuitive UI, it's very easy on the eyes and it genuinely helps me read peacefully after dark, when usually I'd have to avoid blinding white screens. Being able to toggle the overall library and the PDF reader separately is a fantastic feature. Recently the devs announced that all Zotero plugins will need to be rewritten for Zotero 7. This is my favorite plugin and I hope you find time to update it. Thanks for all the great work!

https://forums.zotero.org/discussion/105094/announcing-the-zotero-7-beta/p1

**Comment by tefkah** [comment]
Wow, thank you very much for the kind words and the very generous sponsorship!
I am definitely planning on supporting Zotero 7, but I may have been slightly too ambitious starting out, which is why the migration has taken longer than I'd hoped.

I think to bridge the gap I could create a temporary alpha version of Night that works more like the current plugin, just so people have something to work with. I will try to put up something around this weekend!

**Reply by mechphoenix** [reply]
You're welcome! It's partly on behalf of a group of friends and coworkers who I know all use dark mode, it's really cool that you provide such a big QOL improvement for free. No rush, and I'm very happy to hear Zotero 7 support is in the works :)

**Comment by PenTurDucKenLinE** [comment]
I think this is one of the most important plugins for Zotero 7. Thank you so much for developing past versions! I look forward to the version you're developing for Zotero 7!! Keep up the great work!!!

**Comment by tefkah** [comment]
Hi All!

I just released an alpha version of the plugin that's compatible with Zotero 7 here: https://github.com/tefkah/zotero-night/releases/tag/0.1.0-0

Please check it out and let me know what you think ❤️

**Reply by tefkah** [reply]
@mechphoenix In case you did not see this yet I wanted to mention to you that the plugin is now available for Zotero 7!

**Reply by mechphoenix** [reply]
Thank you so much for your hard work! Keyboard shortcuts and gentle fade transitions are exactly the kind of QOL updates that seem small but make a big difference when you use the plugin every day. And as others on this thread have pointed out, dark mode is one of the most important plugins. Zotero just isn't the same without it. I'm so excited dark mode is available for 7, and thank you very much for making your work available to the community for free!

**Comment by franzbischoff** [comment]
Thank you for this theme!!!!
